resources/Dockerfile.skill

```diff
@@ -70,10 +60,12 @@ COPY --from=overlay /opt/ros/overlay/install /opt/ros/overlay/install
 RUN mkdir /skills \
     && ln -sf /opt/ros/overlay/install/lib/${SKILL_PACKAGE}/${SKILL_NAME}_main /skills/skill_service \
     && ln -sf /opt/ros/overlay/install/share/${SKILL_PACKAGE}/${SKILL_NAME}_skill_config.pbbin /skills/skill_service_config.proto.bin \
-    && sed --in-place --expression '$isource "/opt/ros/overlay/install/setup.bash"' /ros_entrypoint.sh \
-    && sed --in-place --expression '5 a/opt/ros/underlay/install/lib/zenoh_bridge_dds/zenoh_bridge_dds -m client -e tcp/zenoh-router.app-intrinsic-base:7447 --no-multicast-scouting &' /ros_entrypoint.sh
+    && sed --in-place --expression '$isource "/opt/ros/overlay/install/setup.bash"' /ros_entrypoint.sh
 
 LABEL "ai.intrinsic.asset-id"="ai.intrinsic.${SKILL_PACKAGE}.${SKILL_NAME}"
 LABEL "ai.intrinsic.skill-image-name"="${SKILL_NAME}_ros_image"
 
+# TODO(Yadunund): Switch to ZENOH_CONFIG_OVERRIDE after the next jazzy sync.
+ENV ZENOH_SESSION_CONFIG_URI=/opt/zenoh_config.json5
```

Comment:
So, I just "re-learned" today that `CMD` with `["cmd", "arg"]` syntax does not have any of the environment variables set in the docker file because it is not run as any part of the shell, see: https://stackoverflow.com/a/40454758

So, you may need to change the CMD statement or use `sed` to edit the `/ros_entrypoint.sh` like above.